lib/components/result-view/result-view.js

```diff
@@ -28,12 +28,31 @@ class ResultViewComponent extends React.Component {
   containerTooltip = new CompositeDisposable();
   buttonTooltip = new CompositeDisposable();
   expanded: IObservableValue<boolean> = observable(false);
+  ctrlHovering: IObservableValue<boolean> = observable(false);
 
   getAllText = () => {
     if (!this.el) return "";
     return this.el.innerText ? this.el.innerText.trim() : "";
   };
 
+  handleClick = (event: MouseEvent) => {
+    if (this.ctrlHovering.get()) {
```

Comment:
You can check if the `ctrl` key is pressed with `event.ctrlKey` directly. See https://facebook.github.io/react/docs/events.html#mouse-events